packages/cli/src/commands/redis/keyspace-notifications.ts

```diff
@@ -33,7 +33,7 @@ export default class KeyspaceNotifications extends Command {
   }
 
   static args = {
-    database: Args.string(),
+    database: Args.string({description: 'Name of the Redis database. If omitted, it will default to the primary Redis instance associated with the app.'}),
```

Comment:
Change to "name of the Key-Value Store database. If omitted, it defaults to the primary instance associated with the app."